docs/website/package-lock.json

```diff
@@ -9641,9 +9640,9 @@
       }
     },
     "websocket-extensions": {
-      "version": "0.1.3",
-      "resolved": "https://registry.npmjs.org/websocket-extensions/-/websocket-extensions-0.1.3.tgz",
-      "integrity": "sha512-nqHUnMXmBzT0w570r2JpJxfiSD1IzoI+HGVdd3aZ0yNi3ngvQ4jv1dtHt5VGxfI2yj5yqImPhOK4vmIh2xMbGg==",
+      "version": "0.1.4",
+      "resolved": "https://registry.npmjs.org/websocket-extensions/-/websocket-extensions-0.1.4.tgz",
+      "integrity": "sha512-OqedPIGOfsDlo31UNwYbCFMSaO9m9G/0faIHj5/dZFDMFqPTcx6UwqyOy3COEaEOg/9VsGIpdqn62W5KhoKSpg==",
```

Comment:
this works. Note: 
- They changed their license to Apache 2 . 
- we don't depend on this library directly, so bumping in package,json the library that ultimately imports this one might be better.
- Main update this patch does is fix a DOS vulnerability, but we don't use websocket in production (as far as I can tell) so not crucial.

All in all: don't see how it could hurt, but a weird way of updating the packages. (why doesn't dependabot update the package.json?)